src/components/filmDetailsPage/SimilarList.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div class="films-list">
+    <div class="film-card" v-for="film in films" v-bind:key="film.title">
+      <router-link to="/film-details"
+        ><FilmCard
+          :imageSrc="film.image"
```

Comment:
Same as Page.vue

Reply:
In the next PR I moved all the data to the store.